CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 
 * [#8810](https://github.com/rubocop-hq/rubocop/pull/8810): Fix multiple offense detection for Style/RaiseArgs. ([@pbernays][])
 * [#8809](https://github.com/rubocop-hq/rubocop/pull/8809): Fix multiple offense detection for Style/For. ([@pbernays][])
+* [#8799](https://github.com/rubocop-hq/rubocop/pull/8799): Fix `Layouts/SpaceInsideBlockBraces` regression. ([@rdunlop][])
```

Comment:
```suggestion
* [#8799](https://github.com/rubocop-hq/rubocop/issues/8799): Fix `Layouts/SpaceInsideBlockBraces` regression. ([@rdunlop][])
```

Reply:
I hope you don't mind, I noticed that the 8801/8802 line had the same error, so I corrected it too.